src/styles/Home.module.css

```diff
@@ -1,18 +1,18 @@
 .listings_container {
-    padding: 1em;
-    display: grid;
+  padding: 1em;
+  display: grid;
 }
 
 @media (min-width:40rem) {
-    .listings_container {
-        grid-template-columns: repeat(2, 1fr);
-        grid-gap: 1em;
-    }
+  .listings_container {
+      grid-template-columns: repeat(2, 1fr);
+      grid-gap: 1em;
+  }
 }
 
 @media (min-width:60rem) {
-    .listings_container {
-        grid-template-columns: repeat(3, 1fr);
-        grid-gap: 1em;
-    }
-}
+  .listings_container {
```

Comment:
Thought we were going for camelCase not kebab-case or snake_case, could this be fixed?